src/components/links/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { INavLinks } from '../navBar/helper';
+import { LinksWrapper, StyledIconWrapper, StyledLink } from './styled';
+
+export interface LocationType {
+  pathname: string;
+}
+const Links: React.FC<{ links: Array<INavLinks>; location: LocationType }> = ({
+  links,
+  location,
+}) => {
+  return (
+    <>
+      {links.map((link) => (
+        <div key={link.name}>
+          {link.divider}
+          <LinksWrapper active={location.pathname === link.to ? 'true' : ''}>
+            <StyledIconWrapper active={location.pathname === link.to ? 'true' : ''}>
+              {link.icon}
+            </StyledIconWrapper>
+            <StyledLink to={link.to} active={location.pathname === link.to ? 'true' : ''}>
```

Comment:
`location.pathname === link.to ? 'true' : ''` what is the reason to use `? :` ??? `!!location.pathname === link.to` ?

Reply:
or even `location.pathname === link.to`

Reply:
Yes, it will work with the code u mentioned but we will get the red warning in the console : 
" 
react-dom.development.js:86 Warning: Received `false` for a non-boolean attribute `active`.
If you want to write it to the DOM, pass a string instead: active="false" or active={value.toString()}.
If you used to conditionally omit it with active={condition && value}, pass active={condition ? value : undefined} instead.
"

and in order to fix this warning I had to change the type of 'active' to string and have strings as options ('true' : '').

fix: not changed